Building OpenRCT2 on Linux - make install error

### Operating System

Windows 11 64-bit

### OpenRCT2 build

OpenRCT2, v0.4.1 ( 6c1ea78 on develop)

### Describe the issue

After generating the openrct2 binary in the build directory, when I try to run it using `make install` I have the following error :
```
[ 77%] Built target libopenrct2
[ 99%] Built target openrct2
[100%] Built target openrct2-cli
Install the project...
-- Install configuration: ""
[ 77%] Built target libopenrct2
[100%] Built target openrct2
[100%] Generating g2.dat
terminate called after throwing an instance of 'IOException'
  what():  Unable to open '/mnt/c/Users/Pedro\ Lopes/Open\ Source/OpenRCT2/OpenRCT2/resources/g2/sprites.json'
Aborted (core dumped)
make[4]: *** [CMakeFiles/g2.dir/build.make:62: g2.dat] Error 134
make[3]: *** [CMakeFiles/Makefile2:82: CMakeFiles/g2.dir/all] Error 2
make[2]: *** [CMakeFiles/Makefile2:89: CMakeFiles/g2.dir/rule] Error 2
make[1]: *** [Makefile:164: g2] Error 2
CMake Error at cmake_install.cmake:46 (include):
  include called with invalid argument:
  Source/OpenRCT2/OpenRCT2/cmake/download.cmake


make: *** [Makefile:74: install] Error 1

```
Any idea of this could be ? 


### Area(s) with issue?

_No response_

### Steps to reproduce
1. Follow the steps on [https://github.com/OpenRCT2/OpenRCT2/wiki/Building-OpenRCT2-on-Linux](url) until the make install section


### Attachments
<img width="383" alt="image" src="https://user-images.githubusercontent.com/97739376/178159299-55771efb-2fa9-411b-a658-f886ca8f384a.png">

_No response_

```
/usr/bin/cmake -E cmake_progress_start "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build/CMakeFiles" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build/CMakeFiles/progress.marks"
make -f CMakeFiles/Makefile2 all
make[1]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make -f CMakeFiles/libopenrct2.dir/build.make CMakeFiles/libopenrct2.dir/depend
make[2]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
cd "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" && /usr/bin/cmake -E cmake_depends "Unix Makefiles" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build/CMakeFiles/libopenrct2.dir/DependInfo.cmake" --color=
make[2]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make -f CMakeFiles/libopenrct2.dir/build.make CMakeFiles/libopenrct2.dir/build
make[2]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make[2]: Nothing to be done for 'CMakeFiles/libopenrct2.dir/build'.
make[2]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
[ 77%] Built target libopenrct2
make -f CMakeFiles/openrct2.dir/build.make CMakeFiles/openrct2.dir/depend
make[2]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
cd "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" && /usr/bin/cmake -E cmake_depends "Unix Makefiles" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build/CMakeFiles/openrct2.dir/DependInfo.cmake" --color=
make[2]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make -f CMakeFiles/openrct2.dir/build.make CMakeFiles/openrct2.dir/build
make[2]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make[2]: Nothing to be done for 'CMakeFiles/openrct2.dir/build'.
make[2]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
[ 99%] Built target openrct2
make -f CMakeFiles/openrct2-cli.dir/build.make CMakeFiles/openrct2-cli.dir/depend
make[2]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
cd "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" && /usr/bin/cmake -E cmake_depends "Unix Makefiles" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build/CMakeFiles/openrct2-cli.dir/DependInfo.cmake" --color=
make[2]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make -f CMakeFiles/openrct2-cli.dir/build.make CMakeFiles/openrct2-cli.dir/build
make[2]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make[2]: Nothing to be done for 'CMakeFiles/openrct2-cli.dir/build'.
make[2]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
[100%] Built target openrct2-cli
make[1]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
/usr/bin/cmake -E cmake_progress_start "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build/CMakeFiles" 0
make -f CMakeFiles/Makefile2 preinstall
make[1]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make[1]: Nothing to be done for 'preinstall'.
make[1]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
Install the project...
/usr/bin/cmake -P cmake_install.cmake
-- Install configuration: ""
make[1]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
/usr/bin/cmake -S"/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" -B"/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/make -f CMakeFiles/Makefile2 g2
make[2]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
/usr/bin/cmake -S"/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" -B"/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build/CMakeFiles" 99
/usr/bin/make -f CMakeFiles/Makefile2 CMakeFiles/g2.dir/all
make[3]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
/usr/bin/make -f CMakeFiles/libopenrct2.dir/build.make CMakeFiles/libopenrct2.dir/depend
make[4]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
cd "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" && /usr/bin/cmake -E cmake_depends "Unix Makefiles" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build/CMakeFiles/libopenrct2.dir/DependInfo.cmake" --color=
make[4]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
/usr/bin/make -f CMakeFiles/libopenrct2.dir/build.make CMakeFiles/libopenrct2.dir/build
make[4]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make[4]: Nothing to be done for 'CMakeFiles/libopenrct2.dir/build'.
make[4]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
[ 77%] Built target libopenrct2
/usr/bin/make -f CMakeFiles/openrct2.dir/build.make CMakeFiles/openrct2.dir/depend
make[4]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
cd "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" && /usr/bin/cmake -E cmake_depends "Unix Makefiles" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build/CMakeFiles/openrct2.dir/DependInfo.cmake" --color=
make[4]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
/usr/bin/make -f CMakeFiles/openrct2.dir/build.make CMakeFiles/openrct2.dir/build
make[4]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make[4]: Nothing to be done for 'CMakeFiles/openrct2.dir/build'.
make[4]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
[100%] Built target openrct2
/usr/bin/make -f CMakeFiles/g2.dir/build.make CMakeFiles/g2.dir/depend
make[4]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
cd "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" && /usr/bin/cmake -E cmake_depends "Unix Makefiles" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build" "/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build/CMakeFiles/g2.dir/DependInfo.cmake" --color=
make[4]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
/usr/bin/make -f CMakeFiles/g2.dir/build.make CMakeFiles/g2.dir/build
make[4]: Entering directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
[100%] Generating g2.dat
./openrct2-cli sprite build "/mnt/c/Users/Pedro\ Lopes/Open\ Source/OpenRCT2/OpenRCT2/build/g2.dat" "/mnt/c/Users/Pedro\ Lopes/Open\ Source/OpenRCT2/OpenRCT2/resources/g2/sprites.json"
terminate called after throwing an instance of 'IOException'
  what():  Unable to open '/mnt/c/Users/Pedro\ Lopes/Open\ Source/OpenRCT2/OpenRCT2/resources/g2/sprites.json'
Aborted (core dumped)
make[4]: *** [CMakeFiles/g2.dir/build.make:62: g2.dat] Error 134
make[4]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make[3]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make[3]: *** [CMakeFiles/Makefile2:82: CMakeFiles/g2.dir/all] Error 2
make[2]: *** [CMakeFiles/Makefile2:89: CMakeFiles/g2.dir/rule] Error 2
make[2]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make[1]: Leaving directory '/mnt/c/Users/Pedro Lopes/Open Source/OpenRCT2/OpenRCT2/build'
make[1]: *** [Makefile:164: g2] Error 2
CMake Error at cmake_install.cmake:46 (include):
  include called with invalid argument:
  Source/OpenRCT2/OpenRCT2/cmake/download.cmake


make: *** [Makefile:74: install] Error 1
```

I noticed I don't have this directory created ...`"/mnt/c/Users/Pedro\ Lopes/Open\ Source/OpenRCT2/OpenRCT2/build/g2.dat" `...

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -393,7 +393,7 @@ endif ()
 if (NOT (MACOS_BUNDLE AND (NOT CMAKE_OSX_ARCHITECTURES MATCHES "${SYSTEM_MACOS_ARCH}")))
     add_custom_command(
         OUTPUT g2.dat
-        COMMAND ./openrct2-cli sprite build \"${CMAKE_BINARY_DIR}/g2.dat\" \"${ROOT_DIR}/resources/g2/sprites.json\"
+        COMMAND ./openrct2-cli sprite build ${CMAKE_BINARY_DIR}/g2.dat ${ROOT_DIR}/resources/g2/sprites.json
         WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
     )
     add_custom_target(g2 DEPENDS ${PROJECT_NAME} g2.dat)

